src/pages/JobSearch/index.jsx

```diff
@@ -68,48 +73,44 @@ function JobSearch() {
 
   return (
     <Layout>
-      <SearchBar />
+      <SearchBar onSearch={handleSearch} />
       <Flex>
         <Suspense fallback={<Loader />}>
-          {jobs.map(
-            ({
-              id,
-              company,
-              logo,
-              logoBackground,
-              position,
-              postedAt,
-              contract,
-              location,
-            }) => (
-              <FlexItem key={id}>
-                <JobCard
-                  company={company}
-                  logo={logo}
-                  logoBackground={logoBackground}
-                  position={position}
-                  postedAt={postedAt}
-                  contract={contract}
-                  location={location}
-                  jobDetailsURL={`/jobs/${id}`}
-                />
-              </FlexItem>
-            )
-          )}
+          {filteredJobs.length === 0
+            ? 'No results found'
+            : filteredJobs.map(
+                ({
+                  id,
+                  company,
+                  logo,
+                  logoBackground,
+                  position,
+                  postedAt,
+                  contract,
+                  location,
+                }) => (
+                  <FlexItem key={id}>
+                    <JobCard
+                      company={company}
+                      logo={logo}
+                      logoBackground={logoBackground}
+                      position={position}
+                      postedAt={postedAt}
+                      contract={contract}
+                      location={location}
+                      jobDetailsURL={`/jobs/${id}`}
+                      aria-label={`Job at ${company}: ${position}`}
+                    />
+                  </FlexItem>
+                )
+              )}
         </Suspense>
       </Flex>
-
       <InnerBox>
-
-        {cursor < (jobsData.length - 1) &&
-
+        {cursor < jobsData.length - 1 && (
           <Button onClick={handleLoadMore}>Load More</Button>
```

Comment:
Can you also hide the button when the filtered results are no more than 12 job postings? Thank you!